src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -134,21 +134,15 @@ def _snapshot(self, fetcher: ResultFn, loader: ResultFn, *, force_refresh: bool)
             logger.debug(f"[{self.full_name}] fetching {self._table} inventory")
             try:
                 cached_results = list(fetcher())
-                if len(cached_results) > 0:
+                if cached_results:
```

Comment:
Why do you remove the `len` check?

Reply:
It's not needed: an empty list is `False` in a boolean context. (And conversely, a list with any elements is `True`.) This is the [pythonic way](https://peps.python.org/pep-0008/#programming-recommendations):

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false:
```python
# Correct:
if not seq:
if seq:
# Wrong:
if len(seq):
if not len(seq):
```